### PR TITLE
Add Stream Support: Accept-Ranges: bytes for some file extensions

### PR DESCRIPTION
**Pull request by fsutin**
Hey
I have added the extensions to stream by web.config.

<add key="Umbraco.S3.Stream.Extensions" value="mp3,mp4"/>

### CODE REVIEW

File: Umbraco.Storage.S3/Umbraco.Storage.S3/FileSystemVirtualPathProviderFile.cs [hunk 1]
```diff
@@ -1,28 +1,46 @@
...
         private readonly Func<Stream> _stream;
+        private readonly string[] _extensionArray;
 
         public FileSystemVirtualPathProviderFile(string virtualPath, Func<Stream> stream) : base(virtualPath)
         {
             if (stream == null)
                 throw new ArgumentNullException("stream");
             _stream = stream;
+
+            string validExtensions = ConfigurationSettings.AppSettings["Umbraco.S3.Stream.Extensions"];
```
Comment (ElijahGlover) [inline Umbraco.Storage.S3/Umbraco.Storage.S3/FileSystemVirtualPathProviderFile.cs]: "Umbraco.S3.Stream.Extensions" is very generic, please modify the readme file and add this configuration setting for others to fine.

Comment (fsutin) [inline Umbraco.Storage.S3/Umbraco.Storage.S3/FileSystemVirtualPathProviderFile.cs]: What about Umbraco.S3.Stream.ValidFileExtensions ?

**Comment by fsutin** [comment]
I have modified the variable  name and modified the readme file. Please let me know if you need me to do something else.

When would be the next release? Would it include this changes ?

Thank you in advance.

**Comment by fsutin** [comment]
Sorry for bothering you with this @ElijahGlover.
I was wondering if you are going to merge this fix ? Because I'm currently using a patched version of this plugin and I would love to use yours.

**Comment by ElijahGlover** [comment]
This code doesn't belong in the virtual path provider, asp.net request pipeline provides other hooks where this can easily be added.

**Pull request closed without merging**